app/config/dev.exs

```diff
@@ -109,7 +109,13 @@ config :ueberauth, Ueberauth,
 
 if prefix = System.get_env("DEV_PREFIX") do
   config :meadow,
-    dc_api: [v2: %{"base_url" => "https://#{prefix}.dev.rdc.library.northwestern.edu:3002"}]
+    dc_api: [
+      v2: %{
+        "api_token_secret" => "DEVELOPMENT_SECRET",
```

Comment:
Can we add this to secrets manager? I think we have the same value for dev/staging for the API, and I think when the front end starts developing from Meadow it's going to be helpful to have this work and don't want someone accidentally checking in a hard coded value....

Reply:
It's a little tricky because everyone has a different `base_url` value (the API running on their machine), but we share a common secrets manager config. But it should be possible because `Hush` can manipulate values via an anonymous function after retrieval. I'll see if I can make that work quickly.